vcd/resource_vcd_network_routed.go

```diff
@@ -50,35 +50,56 @@ func resourceVcdNetworkRouted() *schema.Resource {
 				Description: "The name of the edge gateway",
 			},
 
-			"netmask": &schema.Schema{
+			"description": &schema.Schema{
 				Type:        schema.TypeString,
 				Optional:    true,
 				ForceNew:    true,
-				Default:     "255.255.255.0",
-				Description: "The netmask for the new network",
+				Description: "Optional description for the network",
+			},
+
+			"interface_type": &schema.Schema{
+				Type:             schema.TypeString,
+				Optional:         true,
+				Default:          "internal",
+				ForceNew:         true,
+				Description:      "Which interface to use (one of `internal`, `subinterface`, `distributed`)",
+				ValidateFunc:     validation.StringInSlice([]string{"internal", "subinterface", "distributed"}, true),
+				DiffSuppressFunc: suppressNetworkUpgradedInterface(),
```

Comment:
As far as I got it it's support back compatibility. So comment would be good regarding that here